src/app/ui/components/containers/headers/dialog-header.tsx

```diff
@@ -1,11 +1,8 @@
 import { ReactNode } from 'react';
```

Comment:
I have moved this full file to the mono-repo in https://github.com/leather-io/mono/pull/252. 

I will replace this file but believe this PR good to merge so we can fix the critical bug of icon placement